platform_plugin_elm_credentials/api/serializers.py

```diff
@@ -0,0 +1,169 @@
+"""Pydantic models for ELMv3 data."""
+from datetime import datetime
+from typing import List, Optional
+from uuid import uuid4
+
+from pydantic import BaseModel, ConfigDict, Field, field_serializer
+
+from platform_plugin_elm_credentials.api.utils import get_current_datetime, to_camel, to_iso_format
+
+
+class QueryParamsModel(BaseModel):
+    """Pydantic model for query parameters."""
```

Comment:
Can we add what the models mean in the docstrings? A short description of what they are.

Reply:
Done